src/objects.rs

```diff
@@ -116,6 +117,12 @@ macro_rules! impl_comment_links {
             fn comment_links_mut(&mut self) -> &mut CommentLinksT {
                 &mut self.comment_links
             }
+            fn add_prefix_on_comment_id(&self, prefix: &str) -> CommentLinksT {
```

Comment:
Instead of this method, I think you should be able to implement the trait `AddPrefix` on `CommentLinksT`.